### PR TITLE
Use a dedicated role for ppl managing their pull secrets

### PR DESCRIPTION
**Pull request by alvaroaleman**

### CODE REVIEW

**Review by hongkailiu — APPROVED** [review]

**Comment by hongkailiu** [comment]
So we do not want to give project admin any more?

**Comment by alvaroaleman** [comment]
@hongkailiu we absolutely do not want to give admin on app.ci

**Comment by stevekuznetsov** [comment]
@alvaroaleman why is this a broader set of permissions than those I found to be necessary for the doc?

File: clusters/app.ci/assets/secrets-manager-role.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - serviceaccounts
+  - secrets
+  verbs:
+  - "*"
+- apiGroups:
+  - image.openshift.io
```
Comment (stevekuznetsov) [inline clusters/app.ci/assets/secrets-manager-role.yaml:17]: Why does the user need any of this?

**Review by stevekuznetsov — COMMENTED** [review]
Please also update the doc https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-get-a-token-for-programmatic-access-to-the-central-ci-registry

**Pull request merged**